apps/api-runtime/src/main/java/com/akto/runtime/policies/AktoPolicyNew.java

```diff
@@ -181,7 +187,7 @@ public void process(HttpResponseParams httpResponseParams) throws Exception {
                     break;
                 case DETERMINE_API_ACCESS_TYPE:
                     try {
-                        saveSample = apiAccessTypePolicy.findApiAccessType(httpResponseParams, apiInfo, filter);
```

Comment:
why?

Reply:
fixed method, it was always returning false earlier, so turned into void, referred avneesh